lib/process-services/src/lib/process-list/components/process-filters.component.ts

```diff
@@ -96,6 +109,10 @@ export class ProcessFiltersComponent implements OnInit, OnChanges {
         }
     }
 
+    isProcessRoute(filter: ProcessInstanceFilterRepresentation) : boolean {
+        return this.currentRoute.includes('process') && this.currentFilter === filter;
```

Comment:
you can do this in the "subcribe" section on the init, and assign to the Boolean property, without calculating each time from the html template